javascript/task1-offline.js

```diff
@@ -0,0 +1,79 @@
+/*
+CHALLENGE: Random Password Generator (Node.js)
+
+OBJECTIVE:
+Build a simple script that generates a random password based on customizable settings.
+
+REQUIREMENTS:
+- Ask the user (or hardcode) the desired password length.
+- Include uppercase, lowercase, numbers, and symbols in the pool.
+- Randomly generate a password of the given length using the pool.
+- Output the password to the terminal.
+
+MUST USE:
+- Arrays and/or strings
+- Functions
+- Math.random()
+- Loops (for or while)
+
+BONUS (optional):
+- Let the user pass in length as a CLI arg (via process.argv)
+- Add options to include/exclude symbols, numbers, etc.
+- Copy password to clipboard (use `clipboardy` if you want to explore packages)
+
+NO EXTERNAL UI, just terminal output.
+*/
+
+passwordLength = 10;
+specialCharacters = ['$', '%', '@', '^', '*', '!']
+
+
+includeNumbers = true
+includeSpecial = true
+
+password = []
+
+for (let i = 0; i < passwordLength; i++) {
+    generated = (Math.random()*100000000000000)%26;
```

Comment:
use let/const here for all new variables.

Also, magic numbers. What is 26?

perhaps `const ALPHABET_CHAR_COUNT = 26`